packages/wonder-blocks-badge/src/components/badge.tsx

```diff
@@ -1,5 +1,202 @@
+import {addStyle, AriaProps} from "@khanacademy/wonder-blocks-core";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import {StyleSheet} from "aphrodite";
 import * as React from "react";
 
-export const Badge = () => {
-    return <div>Badge component</div>;
+type Props = AriaProps & {
+    /**
+     * The kind of badge to display. Defaults to `info`.
+     */
+    kind?: "info" | "success" | "warning" | "critical";
```

Comment:
I started implementing the `gem` kind and am proposing creating separate badge components, see [Figma thread](https://www.figma.com/design/e9qdyiDUBZ3rDabP9S7ulO?node-id=1432-1601#1250633202) for more details. So I might move things around once we decide. It'll be the same logic/styling though!